CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wno-unused-function")
+set(ROCM_EXPORT_PARSE_TARGETS_LIBRARY ON)
```

Comment:
I agree that the default should be ON eventually, but until the PyTorch sets this value, if should be OFF.

Reply:
All right.